src/styles/blueprint-overrides.css

```diff
@@ -52,4 +52,7 @@
   .bp3-dark .bp3-popover .bp3-popover-arrow-fill {
     fill: var(--primary-9-dark);
   }
+  .bp3-tooltip {
```

Comment:
Can you confirm this from Yuen and also check this https://docs.google.com/presentation/d/1vsJr0whxRcRlGYqRUQiT9NMD_fD3ED41I-ufZcCcAxA/edit#slide=id.g10360901e83_0_0

Reply:
Confirmed with Yuen
cc @kapilkumawat86